crates/bevy_text/src/pipeline.rs

```diff
@@ -32,10 +32,22 @@ pub struct CosmicFontSystem(pub cosmic_text::FontSystem);
 
 impl Default for CosmicFontSystem {
     fn default() -> Self {
-        let locale = sys_locale::get_locale().unwrap_or_else(|| String::from("en-US"));
-        let db = cosmic_text::fontdb::Database::new();
-        // TODO: consider using `cosmic_text::FontSystem::new()` (load system fonts by default)
-        Self(cosmic_text::FontSystem::new_with_locale_and_db(locale, db))
+        #[cfg(feature = "system_font")]
+        return {
+            let mut font_system = cosmic_text::FontSystem::new();
+
+            // This is a no-op on unsupported platforms.
+            font_system.db_mut().load_system_fonts();
```

Comment:
I suppose this is the one taking time? Is it possible to run this in an asynchronous way, or in another thread?